src/argparse.lua

```diff
@@ -1204,7 +1314,21 @@ function ElementState:invoke()
    return self.open
 end
 
+function ElementState:check_choices(argument)
+   if self.element._choices then
+      for _, choice in ipairs(self.element._choices) do
+         if argument == choice then
+            return
+         end
+      end
+      local choices_list = "'" .. table.concat(self.element._choices, "', '") .. "'"
```

Comment:
Passing 3 arguments to `table.concat` here.

Reply:
The comma is quoted

Reply:
doh. that's what I get for reviewing in the github UI on a small screen.